src/Swashbuckle.AspNetCore.SwaggerGen/SchemaGenerator/SchemaGenerator.cs

```diff
@@ -195,6 +195,7 @@ private OpenApiSchema GenerateObjectSchema(SerializerMetadata serializerMetadata
 
             foreach (var serializerPropertyMetadata in serializerMetadata.Properties)
             {
+                if (serializerPropertyMetadata.MemberInfo==null) continue;
```

Comment:
Rather than skipping the iteration, I think the logic can be a bit more graceful by continuing with an empty list of custom attributes when `MemberInfo` is null. So, the line below could be replaced with something along the following lines:

```csharp
var customAttributes = (serializerPropertyMetadata.MemberInfo != null)
    ? serializerPropertyMetadata.MemberInfo.GetInlineOrMetadataTypeAttributes()
    : Enumerable.Empty<object>();
```

Reply:
This is where I'd prefer the null check to happen rather than within the extension method

Reply:
I've moved it to schema generator, what's your thought about how it looks now?